response_operations_ui/views/collection_exercise.py

```diff
@@ -13,9 +13,23 @@
 @app.route('/surveys/<short_name>/<period>', methods=['GET'])
 def view_collection_exercise(short_name, period):
     ce_details = collection_exercise_controllers.get_collection_exercise(short_name, period)
+    breadcrumbs = [
```

Comment:
I wonder if there would be a way of refactoring this out in a generic function which you can work out which breadcrumbs we need based on the information we put in i.e. `survey, no period = none active surveys`, `survey and period = active surveys, non active period`. Not needed now but food for thought